session_crypto.py

```
@@ -425,6 +427,189 @@
+            logger.debug("Secure session data cleanup completed")
+            
+        except Exception as e:
+            sanitized_error = sanitize_for_logging(str(e))
+            logger.debug(f"Secure cleanup exception: {sanitized_error}")
+    
+    def __del__(self):
+        """Secure cleanup on object destruction."""
+        try:
+            self.secure_cleanup()
+        except:
+            pass  # Ignore errors during cleanup
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)